guides/common/modules/con_upgrade-paths.adoc

```diff
@@ -2,19 +2,49 @@
 == Upgrade paths
 
 You can upgrade to {ProjectName} {ProjectVersion} from {ProjectName} {ProjectVersionPrevious}.
+The high-level steps in upgrading {Project} to {ProjectVersion} are as follows:
 
 ifdef::satellite[]
-{ProjectServer}s and {SmartProxyServers} on earlier versions must first be upgraded to {Project} {ProjectVersionPrevious}.
+. Ensure that your {ProjectServer}s and {SmartProxyServers} have been upgraded to {Project} {ProjectVersionPrevious}.
 For more information, see {UpgradingPreviousDocURL}[_{UpgradingPreviousDocTitle}_] or {UpgradingDisconnectedPreviousDocURL}[_{UpgradingDisconnectedPreviousDocTitle}_].
 endif::[]
 
-.High-level upgrade steps
-
-The high-level steps in upgrading {Project} to {ProjectVersion} are as follows:
+ifndef::foreman-deb[]
+. Upgrade your {ProjectServer}:
+.. Upgrade your {ProjectServer} to {ProjectVersion}.
+.. Optional: Upgrade the operating system on your {ProjectServer} to {EL}{nbsp}9.
+ifdef::satellite[]
+.. Synchronize the new {ProjectVersion} repositories.
+endif::[]
+. Upgrade your {SmartProxyServers}:
+.. Upgrade all {SmartProxyServers} to {ProjectVersion}.
+.. Optional: Upgrade the operating system on your {SmartProxyServers} to {EL}{nbsp}9.
```

Comment:
Should these link to the specific sections? Then you can drop the "Additional resources" part at the end.

Reply:
I'd rather not. What I dislike about the [3.3 version of Upgrade paths](https://docs.theforeman.org/3.3/Upgrading_and_Updating/index-katello.html#upgrade_paths) is that it basically creates a whole new, but incomplete table of contents for how to upgrade. I'm concerned that if we include links here, users might then follow only the linked sections and skip or overlook the others.

What I'm going for here is a high-level overview without links, just to prepare users for what they should expect in general terms. And then if anyone wants to follow the link in Additional resources, they will be taken to the beginning of the upgrade chapter, which reduces the risk of them skipping parts of the process.

Reply:
However, it might help if I move the link to the introduction. If my goal is to make sure users don't overlook it, it's probably a reasonable thing to do.